src/processTargets/modifiers/scopeHandlers/DocumentScopeHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { Position, Range, TextEditor } from "vscode";
+import { Direction, ScopeType } from "../../../typings/targetDescriptor.types";
+import { getDocumentRange } from "../../../util/rangeUtils";
+import { DocumentTarget } from "../../targets";
+import { OutOfRangeError } from "../targetSequenceUtils";
+import NotHierarchicalScopeError from "./NotHierarchicalScopeError";
+import { TargetScope } from "./scope.types";
+import { ScopeHandler } from "./scopeHandler.types";
+
+export default class DocumentScopeHandler implements ScopeHandler {
+  public readonly scopeType = { type: "line" } as const;
+  public readonly iterationScopeType = { type: "document" } as const;
```

Comment:
Note that iteration scope of document is document, so we get "every file" for free, if for whatever reason someone wants to do that 🤔